languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -116316,8 +116316,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     </token>
                     <marker>
                         <token postag="N.*" postag_regexp="yes">
-                            <exception>famille</exception>
-                            <exception>godot</exception>
+                            <exception postag="[^N].*" postag_regexp="yes"/>
+                            <exception regexp="yes" case_sensitive="yes">[A-ZÉÈÀÙÂÊÎÔÛÄËÏÖÜÇ].*</exception>
+                            <exception regexp="yes">réparation|confirmation|famille|godot|lundi|mardi|mercredi|jeudi|vendredi|samedi|dimanche|janvier|février|mars|avril|mai|juin|juillet|août|septembre|octobre|novembre|décembre|début|mi-.*|fin</exception>
```

Comment:
**suggestion(format)**: maybe here you could use ENTITY like `mois_annee `and `jours_semaine `(but maybe also `unites_temps`, `parties_journee`...) all the ENTITIES are at the top of the grammar.xml file.

You can call them in the rule with structures like:
`<token regexp="yes">&mois_annee;</token>`